modules/ROOT/pages/install/get-started.adoc

```diff
@@ -114,55 +157,60 @@ These can be found in the `couchbase-service-broker` cluster role binding define
 ====
 
 The Deployment creates the Service Broker and ensures it is highly available and a Service makes it discoverable by the Kubernetes Service Catalog in the next step.
-Check the status of the service broker
+
+You can run the following command to check the status of the Service Broker:
 
 [source,console]
 ----
 $ kubectl get deployments
-
+----
+----
 NAME                          	 	READY   	UP-TO-DATE   AVAILABLE   	AGE
 couchbase-operator            		 1/1     		1            	1        	33m
```

Comment:
Just noticed this, and it reminded me to point out that I removed instructions to "install the operator" and went with just the explicit instruction to install the CRD and DAC. So I should probably remove this line?